content/cluster-installation/vmware/ipi-proxy.md

```diff
@@ -1,21 +1,32 @@
 ---
 title: vSphere IPI & Proxy
 linktitle: IPI & Proxy
-description: TBD
-tags:
-  - proxy
-  - VMware
-  - vSphere
+description: Example IPI Install-Configs
+tags: ['Proxy','VMware', 'vSphere', 'IPI']
 ---
+
 # vSphere IPI & Proxy
 
+![targets-and-dependencies](targets-and-dependencies.png)
+
+## Preperations
+
+* Download `oc` client - [HERE](https://access.redhat.com/downloads/content/290/ver=4.17/rhel---9/4.17.9/x86_64/product-software)
+* Download `openshift-installer` - [HERE](https://github.com/openshift/installer/releases)
```

Comment:
I would recommend linking to: https://console.redhat.com/openshift/downloads or straight to https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable/. But console.redhat.com might be better.

Reply:
Good hint.